package.json

```diff
@@ -42,6 +42,7 @@
     "themes:generate": "ts-node --project ./scripts/cli/tsconfig.json ./scripts/cli/generateSassVariableFiles.ts",
     "typecheck": "tsc --noEmit && yarn run packages:typecheck",
     "plugins:build-bundled": "grafana-toolkit plugin:bundle-managed",
+    "monaco:copy": "cp -r ./node_modules/monaco-editor/min/vs ./public/build/monaco",
```

Comment:
maybe:
https://github.com/grafana/grafana/pull/25815/files#diff-80463b1c065d6048b7ea4efcfd4e2ff8f79ef503f461348e57f497496c751f1eR64

Reply:
dodgy wip code - this wasnt actually being used, i did exactly what you linked to :D